tests/bluetooth/audio/ascs/src/test_common.c

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/fff.h>
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/bap.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gatt.h>
+#include <zephyr/bluetooth/uuid.h>
+#include <zephyr/sys/util_macro.h>
+
+#include "bap_unicast_server.h"
+#include "bap_stream.h"
+#include "gatt_expects.h"
+#include "conn.h"
+
+#include "test_common.h"
+
+static uint8_t attr_found(const struct bt_gatt_attr *attr, uint16_t handle, void *user_data)
+{
+	const struct bt_gatt_attr **result = user_data;
+
+	*result = attr;
+
+	return BT_GATT_ITER_STOP;
+}
+
+void test_conn_init(struct bt_conn *conn)
+{
+	conn->index = 0;
+	conn->info.type = BT_CONN_TYPE_LE;
+	conn->info.role = BT_CONN_ROLE_PERIPHERAL;
+	conn->info.state = BT_CONN_STATE_CONNECTED;
+	conn->info.security.level = BT_SECURITY_L3;
```

Comment:
Wouldn't `BT_SECURITY_L2` be more realistic?

Reply:
Ah, that's true